### PR TITLE
#4 Add a Medium Potion

### PR DESCRIPTION
**Pull request by pragpras**
#4 Add a medium potion
PES2UG21CS379
PRAGNA PRASAD

### CODE REVIEW

File: project.py [hunk 1]
```diff
@@ -224,9 +247,9 @@ def shopping():
 
     # If user wants to buy potions
     if int(store1) == 1:
-        print(f"You currently have {potion} Small potions and {ultra_potion} Ultra potions with you.\n")
-        typing("We have 2 types of potion.\n")
-        typing("Small Potion and Ultra Potion\n")
+        print(f"You currently have {potion} Small potions, {ultra_potion} Ultra potions and {medium_potion} Medium potions with you.\n")
```
Comment (Himank101) [inline project.py:250]: medium potion is an unbound local variable

**Review by Himank101 — CHANGES_REQUESTED** [review]
Hey, thanks for contributing!
You have an ubound local error in the project.py file
and you cant access the medium potion mid-battle
please check monster_potion_1() for the same

File: project.py (hunk 1 above)
Comment (pragpras) [reply project.py:250]: I fixed the issue. Could you please close this request so I can resubmit the file with the changes in it?

Comment (pragpras) [reply project.py:250]: The medium potion can be accessed mid-battle now.

**Comment by Himank101** [comment]
!bounty 50

**Comment by Himank101** [comment]
lessgooo! our player will now be safer in the trenches of hell🔥

**Comment by NavinShrinivas** [comment]
!bounty 35

**Pull request merged**